packages/tokens/package.json

```diff
@@ -21,7 +21,7 @@
     "build": "style-dictionary build --config=./style-dictionary.config.js",
     "build:clean": "rm -rf lib/",
     "prepack": "yarn run build",
-    "start": "watch 'yarn run build' ./properties ./node_modules"
+    "start": "watch 'yarn run build' ./properties"
```

Comment:
There's no longer a node_modules directory that's local to the eds-tokens package, so this command was broken.

I believe most folks would restart any running processes when installing dependencies anyway, so we don't need to watch the node_modules directory.